File: vmc/beginning/l 2.3/rocket.c
#include<stdio.h>

int make_head(int n){
    int i,j ;
    for(j=1; j<= n; j++){
        for(i=1 ; i<= n-j+1 ; i++)
            printf(" ");
        if(j==1)
        {
            printf("^");
        }
        else
        {
            for(int k=1; k<j ; k++)
                printf("/");
            printf("|");
            for(int k=1; k<j ; k++)
                printf("\\");   
            }
            printf("\n");
        }
        return 0;
}
int make_seperator(int n)
{
    printf("+");
    for(int i=1; i<=(n-2);i++)
    {
        printf("-");
    }
    printf("+\n");
}
int make_body(int n)
{
    int i,j ;
    for(j=1;j<=4;j++)
    {
        printf("+");
        for(int i=1; i<=(n-2);i++)
        {
            printf("*");
        }
        printf("+\n");
    }
}

int main()
{
    make_head(4);
    make_seperator(9);
    make_body(9);
    make_seperator(9);
    make_body(9);
    make_seperator(9);
    make_head(4);
}

File: vm/beginning/l 2.2/rocket.py
def make_head(w):
    center = int((w+1)/2)
    print(" "*(center-1)+"^")
    for i in range(1,center-1):
        print(" "*(center-i-1)+"/"*i+"|"+"\\"*i)

def make_separator(w):
    print("+"+"-"*(w-2)+"+")


def make_body(w,l):
    for i in range(l):
        print("+"+"*"*(w-2)+"+")
    make_separator(w)





def make_rocket(w,c,l):
    make_head(w)
    make_separator(w)

    for i in range(c):
        make_body(w,l)

    make_head(w)


make_rocket(9,2,5)
